packages/core/components/Modal/CodeSnippet/ConvertToZarr.tsx

```diff
@@ -0,0 +1,307 @@
+                                        onClick={() => setOpts((o) => ({ ...o, destination: "" }))}
+                                    >
+                                        ×
+                                    </button>
+                                )}
+                            </div>
+                        </div>
+
+                        <SecondaryButton
+                            text="BROWSE"
+                            onClick={() => {
+                                // what should go here?
+                            }}
+                            className={styles.browseBtn}
+                        />
+                    </div>
+
+                    <div className={styles.section} style={{ marginTop: 20 }}>
+                        <div className={styles.label}>Scenes</div>
+                        <ChoiceGroup
+                            className={styles.choiceGroup}
+                            defaultSelectedKey={opts.scenesMode}
+                            options={[
+                                { key: "all", text: "All scenes" },
+                                { key: "single", text: "Single scene" },
+                            ]}
+                            onChange={(_, option) =>
+                                setOpts((o) => ({
+                                    ...o,
+                                    scenesMode: (option?.key as "all" | "single") ?? "all",
+                                }))
+                            }
+                        />
+
+                        {opts.scenesMode === "single" && (
+                            <div className={styles.searchBox}>
+                                <TextField
+                                    value={opts.sceneIndex}
+                                    type="number"
+                                    onChange={(_e, v) =>
+                                        setOpts((o) => ({
+                                            ...o,
+                                            sceneIndex: (v ?? "0").replace(/[^\d]/g, ""),
+                                        }))
+                                    }
+                                    placeholder="0"
+                                    borderless
+                                />
+                            </div>
+                        )}
+                    </div>
+                </div>
+
+                {/* RIGHT COLUMN*/}
+                <div className={styles.rightCol}>
+                    <Snippet title="Setup" text={snippet?.setup} copyAria="Copy setup" />
+                    <Snippet title="Code" text={snippet?.code} copyAria="Copy code" />
+                </div>
+            </div>
+        </div>
+    );
+
+    const footer = (
+        <div className={styles.footerButtons}>
+            <SecondaryButton className={styles.closeButton} onClick={onDismiss} text="CLOSE" />
+        </div>
+    );
+
+    return (
+        <BaseModal
```

Comment:
Noting that this modal doesn't use the CodeSnippet wrapper or any styling from other components in this directory. could be slightly organizationally confusing, but I also don't have a strong preference about it since they are thematically related